services/transaction_record.proto

```diff
@@ -122,4 +122,22 @@ message TransactionRecord {
      * transaction with a (previously unused) alias, the new account's alias. 
      */
      bytes alias = 16;
+
+     /**
+      * The current balances of any adjusted crypto allowances as a result of this
+      * transaction.
```

Comment:
Maybe specify only populated in the case of a `CryptoAdjustAllowanceTransaction`

Reply:
Done